libs/agno/agno/team/team.py

```diff
@@ -685,7 +696,23 @@ def run(
                     _tools.append(self.get_member_information)
 
             self._add_tools_to_model(self.model, tools=_tools)  # type: ignore
-
+            for member in self.members:
```

Comment:
We shouldn't do this. When we register a team on the API side, we should also register the agents in the team. So the team registration should have all the needed information.